7.1 дз/7.1 дз/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Homework
+{
+
+    class Program
+    {
+
+        public bool CountNum(double m, double n)
```

Comment:
Неверный тип переменных. Задание:

> Написать метод, который вычисляет значение логического выражения в
зависимости от значений **целочисленных** аргументов.

Вы выбрали тип чисел с плавающей точкой. Нужно было так:
```suggestion
        public bool CountNum(int m, int n)
```